<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportFrame</class>
 <widget class="QFrame" name="ReportFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="SearchLineEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search in document&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="ClearSearchButton">
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Programming/Eclipse/DegenPrimerGUI/resources/DegenPrimerUI.qrc">
       <normaloff>:/clear.svg</normaloff>:/clear.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="ReportTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SearchLineEdit</tabstop>
  <tabstop>ReportTextEdit</tabstop>
  <tabstop>ClearSearchButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Programming/Eclipse/DegenPrimerGUI/resources/DegenPrimerUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ClearSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
